definitions/divorce/json/CaseEvent.json

```diff
@@ -1068,6 +1070,7 @@
     "PostConditionState": "SOTAgreementPayAndSubmitRequired",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-create",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/solicitor-created",
```

Comment:
this was added by @alexwakeman on 04/07. 
commit 1f22efbd08dfccc8708d40d15cc84dc6641b20bb

I don't know if we actually need both urls as they are the same...

Reply:
@hosainnet do you think this ^^ is ok?

Reply:
One is -create and the other is -created (the notifications enpdoint) I think?

Reply:
I don't think its merged yet, https://github.com/hmcts/div-case-orchestration-service/pull/357

Reply:
Look at master, it's in:
https://github.com/hmcts/div-ccd-definitions/blob/master/definitions/divorce/json/CaseEvent.json#L1075

Shall I add it to prod in my PR or remove this line as the feature seems not to be prod-ready.

Reply:
I will remove it from this PR